src/flowTypes.js

```diff
@@ -105,14 +105,25 @@ export type BoxItemPermission = {
 };
 
 export type User = {
-    avatarUrl: string,
-    email: string,
+    avatarUrl: ?string,
+    email: ?string,
```

Comment:
dont think the user object has anything called an email
and its avatar_url per the docs

probably ? should come before : if the field is optional

Reply:
Changed it to avatar_url and moved the ?.
The reason email was included here is because BRUI expects this format for @mention selectors. The response from the endpoint is modified to use email instead of login for the user in the current ActivityFeed. 
```
type DefaultSelectorRowProps = {
    item?: {
        name?: string,
        email?: string,
    },
};
```

Would it make sense to do something like this for our case? 
```
export type User = {
    avatar_url?: string,
    email?: string,
    id: string,
    login: string,
    name: string,
    type: 'user'
};

export type Collaborator = {
    id: string,
    name: string,
    item: User,
}
```

Reply:
Or would you recommend 
```
export type CollabItem = {
    email: string,
    id: string,
    name: string,
};

export type Collaborator = {
    id: string,
    name: string,
    item: CollabItem,
}
```

Reply:
Scratch all of that. I think I can just use SelectorItem